spkrepo/views/frontend.py

```diff
@@ -96,9 +96,9 @@ def packages():
 
 @frontend.route("/package/<name>")
 def package(name):
-    # TODO: show only packages with at least a version and an active build
+    # show only packages with at least a version and an active build
     package = Package.query.filter_by(name=name).first()
-    if package is None:
+    if package is None or not package.any_builds_active():
```

Comment:
```suggestion
    if package is None or not package.any_builds_active:
```

I think this is the correct syntax.

Were you able to test locally?

Reply:
Sorry, I don't have a local setup to test with.